test/scenario/metering-plans/standard-services-hours/src/test/test.js

```diff
@@ -0,0 +1,171 @@
+'use strict';
+
+const moment = require('abacus-moment');
+const oauth = require('abacus-oauth');
+const request = require('abacus-request');
+const util = require('util');
+const uuid = require('uuid');
+
+const debug = require('abacus-debug')('abacus-service-hours-scenario-test');
+
+const doGet = util.promisify(request.get);
+const doPost = util.promisify(request.post);
+
+const env = {
+  secured: process.env.SECURED === 'true',
+  systemClientId: process.env.CLIENT_ID,
+  systemClientSecret: process.env.CLIENT_SECRET,
+  authServerURL: process.env.AUTH_SERVER_URL || 'http://localhost:9882',
+  collectorURL: process.env.COLLECTOR_URL || 'http://localhost:9080',
+  reportingURL: process.env.REPORTING_URL || 'http://localhost:9088',
+  provisioningURL: process.env.PROVISIONING_URL || 'http://localhost:9880',
+  pollInterval: process.env.POLL_INTERVAL || 300
+};
+
+describe('standard services hours scenario test', () => {
+  const timestamp = moment.now();
+  const hourInMillis = 1 * 60 * 60000;
```

Comment:
If you are doing it with a formula, expand it all the way: `60 * 60 * 1000`. The leading `1` is optional.